.changeset/nice-icons-give.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-backend-node': minor
```

Comment:
`plugin-catalog-backend` started to be split into multiple packages to keep the dependency tree cleaner:
![image](https://user-images.githubusercontent.com/720821/126608907-009e77b9-c473-4241-a36b-314d576c73de.png)

How about introducing a `@backstage/plugin-search-backend-module-elasticsearch`?

Reply:
Just to note that we'll introduce either a breaking change or duplicated types or a circular dependency with this swap. I think it's worth doing regardless, opting for a breaking change.

Reply:
Actually we can pipe type movements through, so no breakage needed.